__tests__/topics.test.js

```diff
@@ -71,3 +71,45 @@ describe('CORE: GET /api', () => {
       });
   });
 });
+
+describe('CORE: GET /api/articles/:article_id', () => {
+  test('200: Should return an article object', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        expect(typeof body.article).toBe('object');
+      });
+  });
+  test('200: Returning object should have the appropriate keys', () => {
+    return request(app)
+      .get('/api/articles/1')
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.article).toHaveProperty('author');
```

Comment:
I'd recommend using a more flexible jest matcher for this eg **expect.objectContaining** or **.toMatchObject** this will allow for later extension of the endpoint and will make your test more robust 👍